config/tcs_config.toml

```diff
@@ -40,7 +40,8 @@ max_work_order_count = 10
 # Service Provider ID (SPID) is a 32-digit hex string tied to the
 # enclave implementation. Replace dummy SPID with value obtained after
 # subscription to run TCS in Intel SGX HW mode.
-spid = "DEADBEEF00000000DEADBEEF00000000"
+#spid = "DEADBEEF00000000DEADBEEF00000000"
+spid = "F96B1426FEC20F217482D3C9C31FC7EA"
```

Comment:
remove the unwanted commented code. and why there is change in spid?

Reply:
The `spid` is just an example that needs to be changed.  Do not put a real service provider ID here.

Reply:
I will keep the original dummy value and delete real spid value.